src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,17 @@
-import React from 'react';
```

Comment:
You should import React since you are using JSX syntax in this file. Add `import React from 'react';` at the beginning of the file.